templates/Dockerfile

```
@@ -0,0 +1,33 @@
+# This Dockerfile is based on the example automation Dockerfile available here:
+# https://github.com/algorand/generator/blob/master/examples/repo_template_dir/Dockerfile
+
+# ==============================
+# > BUILD AND GENERATE
+# ==============================
+
+FROM algorand-generator as generator
+
+# ==============================
+# > FORMAT
+# ==============================
+
+FROM node:latest as formatter
+
+# Copy files from previous build stage
+COPY --from=generator /repo /repo
+WORKDIR /repo
+
+# Install dependencies
+RUN npm install
```

```suggestion
RUN npm ci
```
This would probably be better so that no unexpected package updates happen.

---

Good idea! I'll commit your suggestion.